File: HackerEarth Algo Problem/Basics of InputOutput/7-Toggle String/TestClass.py
#include<stdio.h>
#include<string.h>
int main()
{

        String data=sc.nextLine();
		char data[100];
		gets(data);
      //  System.out.println(data);
        
       // String out="";
		char out[100];
        //String temp="";
		char temp[]=out;
		
		
        for(int i=0;i<strlen(data);i++)
        {
            if(data[i]>='a'&&data[i]<='z')
            {
                
				
				
				temp[i]=data.charAt(i)+"";
				
                out+=temp.toUpperCase();
                
            }
            else if(data.charAt(i)>='A'&&data.charAt(i)<='Z')
            {
                temp=data.charAt(i)+"";
                out+=temp.toLowerCase();
                
            }
            else
            {
                out+=data.charAt(i)+"";
            }
        }
        System.out.println(out);
return 0;
}






import java.util.Scanner;
public class TestClass
{
    public static void main(String[] args) {
        
        Scanner sc=new Scanner(System.in);
        String data=sc.nextLine();
      //  System.out.println(data);
        
        String out="";
        String temp="";
        for(int i=0;i<data.length();i++)
        {
            if(data.charAt(i)>='a'&&data.charAt(i)<='z')
            {
                temp=data.charAt(i)+"";
                out+=temp.toUpperCase();
                
            }
            else if(data.charAt(i)>='A'&&data.charAt(i)<='Z')
            {
                temp=data.charAt(i)+"";
                out+=temp.toLowerCase();
                
            }
            else
            {
                out+=data.charAt(i)+"";
            }
        }
        System.out.println(out);
    }
}

File: HackerEarth Algo Problem/Basics of InputOutput/6-Palindromic String/TestClass.java
import java.util.Scanner;
class TestClass {
    public static void main(String args[] ) throws Exception {
       Scanner sc=new Scanner(System.in);
       String s=sc.nextLine();

//String s="";
//s=sc.nextLine();

char str[]=s.toCharArray();
int j=0,c=0;
int l=s.length();
  for(int i=l-1;i>=0;i--)
  {
	  
	  if(str[j]==str[i])
	  {
		  c++;
	  }
	  j++;
	  
  }
   if(c==l)
  {
	  System.out.println("YES");
  }
  else
  {
	System.out.println("NO");
  }

       

    }
   
    }
